db/1.10/modules/ROOT/pages/database-connector-connection.adoc

```diff
@@ -61,6 +61,84 @@ In the XML editor, both the spring and database configurations look like this:
 </db:config>
 ----
 
+== Configure a Data Source Reference Connection when in XA Transaction Mode
+
+If your Mule app uses Database Connector operations such as *Insert*, *Select* or others that run with an XA transaction, and connection pooling is enabled via the *Pooling profile* fields of the connector configuration, the default connection pool implementation for the Data Source Reference Connection is Enhydra XAPool.
```

Comment:
I'd say "If your Mule App requires operations bound to XA transactions, and the connection pooling is enabled for the database configuration in use .."